firmware/dev/f4dev/src/cubemx/Src/main.c

```diff
@@ -531,9 +533,14 @@ void StartDefaultTask(void *argument)
     /* init code for USB_DEVICE */
     MX_USB_DEVICE_Init();
     /* USER CODE BEGIN 5 */
+    //add the usb examples here:
+
+
+    
     UART modem_uart = { .handle = &huart2 };
+
     /* Infinite loop */
-    // uint8_t message[7] = { 66, 79, 79, 66, 83, 13, 10 };
```

Comment:
Let's undo this diff too! I think it's old telem testing code but no reason to diff it in this PR